### ISSUE TITLE
Concerning about the pretrained PWC-Net model for pytorch

### ISSUE THREAD

**Issue by KSJhon** [csbhr/CDVD-TSP#17]
Dear!
I am trying to train CDVD-TSP on my own database, but i don't know where i can download the pretrained PWC-Net model.
I found that downloaded from https://github.com/NVlabs/PWC-Net/tree/master/PyTorch is corrupted.
Thank you in advance.

**Comment by csbhr** [comment]
You can download the pretrained PWC-Net from the following link:
https://drive.google.com/drive/folders/1lw_1jITafEQ9DvMys_S6aYwtNApYKWsz?usp=sharing

**Comment by KSJhon** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    Thank your for your guidance, but my issue is related to training, not for test.Sorry for taking your time.
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                quanjx046
                            
                            
                                
                            
                    
                        
                            
                                    quanjx046@nenu.edu.cn
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 12/30/2020 11:36，Haoran Bai<notifications@github.com> wrote： 



You can download the pretrained PWC-Net from the following link:
https://drive.google.com/drive/folders/1lw_1jITafEQ9DvMys_S6aYwtNApYKWsz?usp=sharing

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by csbhr** [comment]
The pre-trained PWC-Net model used in the training process is also included in this link:

https://drive.google.com/drive/folders/1lw_1jITafEQ9DvMys_S6aYwtNApYKWsz?usp=sharing

**Comment by ghost** [comment]
> font{ line-height: 1.6; } ul,ol{ padding-left: 20px; list-style-position: inside; } Thank your for your guidance, but my issue is related to training, not for test.Sorry for taking your time. quanjx046 quanjx046@nenu.edu.cn 签名由 网易邮箱大师 定制 On 12/30/2020 11:36，Haoran Bai<notifications@github.com> wrote： You can download the pretrained PWC-Net from the following link: https://drive.google.com/drive/folders/1lw_1jITafEQ9DvMys_S6aYwtNApYKWsz?usp=sharing —You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

so how to download the PWC-Net Pretrained model if I want to train CDVD-TSP?Have you solved this problem?Thanks a lot.
